apps/src/code-studio/reporting.js

```diff
@@ -48,7 +48,8 @@ reporting.sendReport = function (report) {
       queryItems.push(key + '=' + report[key]);
     }
   }
-  var queryString = queryItems.join('&');
+
+  var queryString = encodeURI(queryItems.join('&'));
```

Comment:
I think you want to encode each item in `queryItems` before joining?  I believe we should use `encodeURIComponent` instead.